qa/tasks/ses_qa.py

```diff
@@ -178,6 +179,123 @@ def _get_drive_group_limit(self, **kwargs):
                            if 'deepsea' in x and 'drive_group' in x['deepsea'])
         return int(drive_group['deepsea']['drive_group']['custom']['data_devices']['limit'])
 
+    def ses_rack_dc_region_unavailability(self, **kwargs):
+        """
+        Simulates rack, DC and region unavailability by 
+        modifying Ceph crushmap
+        """
+        self.scripts.run(
+                self.master_remote,
+                'ses_rack_dc_region_unavailability.sh',
+                )
+    
+    def ses_network_failure(self, **kwargs):
+        """
+        Simulates network failure using tc netem tool
+        """
+        self.scripts.run(
+                self.master_remote,
+                'ses_network_failure.sh',
+                )
+
+    def ses_cephfs_test_mount(self, **kwargs):
+        self.scripts.run(
+                self.master_remote,
+                'ses_cephfs_test_mount.sh',
+                args=self.nodes_monitor,
+                )
+
+    def ses_ceph_osd_tiering(self, **kwargs):
+        self.scripts.run(
+                self.master_remote,
+                'ses_ceph_osd_tiering.sh',
+                )
+
+    def ses_disk_fault_injection(self, **kwargs):
+        self.scripts.run(
+                self.master_remote,
+                'ses_disk_fault_injection.sh',
```

Comment:
here, you want to pass the hostname of a storage node, right?

Reply:
right, this is not finished yet.